### PR TITLE
Update dependencies and clean-up

### PR DESCRIPTION
**Pull request by imobach**
Updating dependencies and doing some clean-up.

### CODE REVIEW

File: .jshintrc [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "globalstrict": true,
   "bitwise": true,
-  "camelcase": true,
```
Comment (jreidinger) [inline .jshintrc]: why no longer want camelcase?

File: bower.json [hunk 2]
```diff
@@ -3,30 +3,30 @@
...
+    "ngEmbed": "ng-embed#~2.0.3"
   },
   "devDependencies": {
-    "angular-mocks": "~1.3.4"
+    "angular-mocks": "~1.4.8"
   },
   "resolutions": {
     "jquery": "~2.1.1",
-    "angular": "~1.3.4"
+    "angular": "~1.4.8"
```
Comment (jreidinger) [inline bower.json]: brave change, but if everything works :)

Comment (imobach) [inline bower.json]: I've made some tests and it seems to work (TM) :)

File: .jshintrc (hunk 1 above)
Comment (imobach) [inline .jshintrc]: This options is [deprecated](http://jshint.com/docs/options/#camelcase) and it's always complaining about some Janus code. Hopefully we could use [JSCS](http://jscs.info/) in the near future for styling issues.

**Comment by ancorgs** [comment]
Let's merge and see what happens :-)

**Comment by imobach** [comment]
To be sure, you can do:

```sh
rm -r bower_components
bower install
```

**Pull request merged**